tests/lib/rules/no-undef-properties.js

```diff
@@ -561,7 +561,248 @@ tester.run('no-undef-properties', rule, {
         }
       }
     },
+    {
+      // Vuex
+      filename: 'test.vue',
+      code: `
+      <script>
+      import { mapState } from 'Vuex';
```

Comment:
Please change `Vuex` to `vuex` in all test cases.

Reply:
Good catch. Will update!